modules/zenith-public/lua/1-enum/zxi/mobHelpers.lua

```diff
@@ -22,4 +22,51 @@ zxi.mobHelpers.getRespawnVarName = function(mob, mobId)
     return fmt(template, mob:getName(), mob:getID())
 end
 
+zxi.mobHelpers.zoneInitReposition = function(zone, mobList)
+    if not zone or not mobList then
+        return
+    end
+
+    for mobName, cfg in pairs(mobList) do
+        -- query all spawns of this mob in the zone
+        local entities = zone:queryEntitiesByName(mobName)
+
+        if entities and #entities > 0 then
+            -- if limit is zero or nil treat as no spawns allowed
+            if cfg.limit and cfg.limit <= 0 then
```

Comment:
technically this is "if limit is defined as negative or zero"

Probably simpler to just check explitily for 0

```suggestion
            -- if limit is zero disable all spawns for given mobName
            if cfg.limit and cfg.limit == 0 then
```